src/Core/Scheduler/Providers/WMTS_Provider.js

```diff
@@ -85,18 +85,19 @@ WMTS_Provider.prototype.getXbilTexture = function getXbilTexture(tile, layer, ta
     const url = this.url(coordWMTS, layer);
 
     return OGCWebServiceHelper.getXBilTextureByUrl(url, layer.networkOptions).then((texture) => {
-        const { min, max } = OGCWebServiceHelper.ioDXBIL.computeMinMaxElevation(
-            texture.image.data,
-            SIZE_TEXTURE_TILE, SIZE_TEXTURE_TILE,
-            pitch);
+        const { min, max } = (texture.min == undefined && texture.max == undefined) ? texture :
+            OGCWebServiceHelper.ioDXBIL.computeMinMaxElevation(
+                texture.image.data,
+                SIZE_TEXTURE_TILE, SIZE_TEXTURE_TILE,
+                pitch);
 
         texture.coords = coordWMTS;
 
         return {
             texture,
             pitch,
-            min: min === undefined ? 0 : min,
-            max: max === undefined ? 0 : max,
+            min: !min ? 0 : min,
+            max: !max ? 0 : max,
```

Comment:
If there is no value at this point, it means there is a problem with the array (there is no buffer basically). Which means that returning 0 in min and max is not a solution. Imo this is an error if it happens, and we should throw somewhere (in parseXBil maybe). WDYT?

Reply:
It should be studied why this case happens and what is the impact itowns. In any case it isn't the scope of the PR